src/index.ts

```diff
@@ -73,6 +81,12 @@ export function datadog<TEvent, TResult>(
   const traceListener = new TraceListener(finalConfig, handlerName);
   const listeners = [metricsListener, traceListener];
 
+  // Only wrap the handler once unless forced
```

Comment:
This is my bad, but I just realized this might be kind of fragile, if we have two handlers in a single file, I think this will break.

eg.
```
const firstFunction = datadatog(() => {..});
const secondFunction = datadatog(() => {..}); // This won't get wrapped, customer would have to use forceWrap
```

I think you might have to do some fancy logic inside the bottom lambdas